gno.land/pkg/sdk/vm/handler.go

```diff
@@ -28,6 +29,8 @@ func (vh vmHandler) Process(ctx sdk.Context, msg std.Msg) sdk.Result {
 		return vh.handleMsgCall(ctx, msg)
 	case MsgRun:
 		return vh.handleMsgRun(ctx, msg)
+	case MsgEval:
+		return vh.handleMsgEval(ctx, msg)
```

Comment:
```suggestion
```

Having a `MsgEval` struct instead of using arguments may be sensible. However, I believe we should not register `Eval` as a possible transaction type in that manner.